actions/adoptium-dependency/main.go

```diff
@@ -70,16 +70,18 @@ func main() {
 	}
 
 	versions := make(actions.Versions)
+	additionalOutputs := make(actions.Outputs)
 	for _, r := range raw {
 		versions[strings.ReplaceAll(r.VersionData.Semver, "+", "-")] = r.Binaries[0].Package.Link
+		additionalOutputs["source"] = r.Source.Link
```

Comment:
Unless I'm missing something - the source URI here is not linked to the release/asset version that we are looping. This results in a source URI that doesnt match the latest version.

In other actions, I used the looping version as the key to the source map. Then when we calculate Latest, we use that value to pull the corresponding source from the map. See https://github.com/paketo-buildpacks/pipeline-builder/blob/fea2141ee5a3665301b3ec35636ff35984de37fd/actions/tomcat-dependency/main.go#L52-L53

Reply:
oups, yes indeed!